src/ralph/data_importer/resources.py

```diff
@@ -240,7 +248,7 @@ class DiskShareMountResource(resources.ModelResource):
     share = fields.Field(
         column_name='share',
         attribute='share',
-        widget=widgets.ForeignKeyWidget(DiskShare, 'wwn'),
+        widget=widgets.ForeignKeyWidget(ImportedForeignKeyWidget),
```

Comment:
it's by purpose or a mistake? Shouldn't it be `widget=ImportedForeignKeyWidget(DiskShare),`?